MantidQt/MantidWidgets/src/MuonFitPropertyBrowser.cpp

```diff
@@ -59,6 +59,10 @@
 
 namespace {
 Mantid::Kernel::Logger g_log("MuonFitPropertyBrowser");
+const std::string CUSTOM_LABEL{"Custom"};
```

Comment:
Why not construct them directly as `QString`s to save the overhead in always converting them below?